src/documentation/pages/components/forms/search-input/examples/index.vue

```diff
@@ -0,0 +1,29 @@
+<template lang="pug">
+  div
+    h2 Examples
+
+    <Base />
+    <Disabled />
+    div(v-if="$store.state.themes.theme === 'portal'")
+      <SizesPortal />
+    div(v-else)
+      <SizesLumenCenturyLink />
```

Comment:
instead of using divs you could add the conditionals to the components themselves, e.g.:

```
      <SizesPortal v-if="$store.state.themes.theme === 'portal'" />
      <SizesLumenCenturyLink v-else />
```